docs/src/gui/gmoccapy.adoc

```diff
@@ -1178,8 +1178,7 @@ The page is separated in three main tabs:
 
 On this tab you will find the following options:
 
-==== Main Window
-
+.Main Window
```

Comment:
Can you exclude these changes from this PR? I am currently working on the gmoccapy doc and also reordering the sections to fit the max. level 3.

Reply:
Argh. I only read this now after the rebase. Please just submit your work and I will then go through the conflicts?

Reply:
>Argh. I only read this now after the rebase. Please just submit your work and I will then go through the conflicts?

No worries, I pushed my changes to the gmoccapy docs yesterday =)